go.mod

```diff
@@ -4,6 +4,7 @@ go 1.15
 
 require (
 	github.com/aws/aws-sdk-go v1.33.1
+	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
```

Comment:
@brycahta do we need to add to the third party licenses for this?

Reply:
Yeah, nice catch.

Should be added [here](https://github.com/aws/aws-node-termination-handler/blob/main/THIRD_PARTY_LICENSES#L44) following similar format

Reply:
Done.